src/libcrun/handlers/wasmedge.c

```diff
@@ -183,7 +183,23 @@ libwasmedge_configure_container (void *cookie arg_unused, enum handler_configure
   int ret;
   runtime_spec_schema_config_schema *def = container->container_def;
 
-  if (getenv ("WASMEDGE_PLUGIN_PATH") == NULL && getenv ("WASMEDGE_WASINN_PRELOAD") == NULL)
+  char **container_env = def->process->env;
+  const char *plugin_path = NULL;
+  const char *preload = NULL;
+
+  for (char **env = container_env; env && *env; env++)
+    {
+      if (strncmp (*env, "WASMEDGE_PLUGIN_PATH=", 21) == 0)
+        {
+          plugin_path = *env + 21;
```

Comment:
nit: no need for brackets for a single statement

Reply:
Fixed